rconweb/api/votemap.py

```diff
@@ -122,3 +127,158 @@ def reset_votemap_state(request):
         },
         command="reset_votemap_state",
     )
+
+
+@csrf_exempt
+@login_required(True)
+def get_map_whitelist(request):
+    v = VoteMap()
+    return api_response(
+        failed=False,
+        result=[map for map in v.get_map_whitelist()],
+        command="get_map_whitelist",
+    )
+
+
+@csrf_exempt
+@login_required(True)
+@record_audit
+def do_add_map_to_whitelist(request):
+    data = _get_data(request)
+    try:
+        map_name = data["map_name"]
+    except KeyError:
+        return api_response(
+            failed=True,
+            result=None,
+            command="do_add_map_to_whitelist",
+        )
+
+    v = VoteMap()
+    audit("do_add_map_to_whitelist", request, {"map_name": map_name})
+    v.do_add_map_to_whitelist(map_name=map_name)
+
+    return api_response(
+        failed=False,
+        result=None,
+        command="do_add_map_to_whitelist",
+    )
+
+
+@csrf_exempt
+@login_required(True)
+@record_audit
+def do_add_maps_to_whitelist(request):
+    data = _get_data(request)
+
+    try:
+        map_names = data["map_names"]
+    except KeyError:
+        return api_response(
+            failed=True,
+            result=None,
```

Comment:
Same here, would need basic error message. This applies to more functions further down it seems.